modules/appeals_api/app/mailers/appeals_api/weekly_error_report_mailer.rb

```diff
@@ -25,8 +25,8 @@ def build(recipients:, friendly_duration: '')
 
     def stuck_err_hlrs
       stuck_hlr_statuses = HigherLevelReview::STATUSES - HigherLevelReview::COMPLETE_STATUSES - ['error']
-      err_hlrs   = HigherLevelReview.v2.where(status: 'error')
-      stuck_hlrs = HigherLevelReview.v2.where(status: stuck_hlr_statuses)
+      err_hlrs   = HigherLevelReview.v2_or_v0.where(status: 'error')
+      stuck_hlrs = HigherLevelReview.v2_or_v0.where(status: stuck_hlr_statuses)
```

Comment:
I anticipate that we'll be making more changes to reporting in order to report V2 and V0 appeals separately, but for now, I'm including V0 appeals in all existing reporting.